bears/ruby/RuboCopBear.py

```diff
@@ -28,8 +28,186 @@ def create_arguments(filename, file, config_file, rubocop_config: str=""):
+            should be inspected?
+            options: 'always_inspect', 'always_ignore', 'ignore_implicit',
+                     'ignore_explicit'
+        :param align_parameters:
+            Alignment of parameters in multi-line method calls.
+
+            The `with_first_parameter` style aligns the following lines along
+            the same column as the first parameter.
+
+            The `with_fixed_indentation` style aligns the following lines
+            with one level of indentation relative to the start of the line
+            with the method call.
+        :param class_check:
+            How to check type of class.
+            options: 'is_a?', 'kind_of?'
+        :param comment_keywords:
+            Checks formatting of special comments.
+        :param min_if_unless_lines:
+            Defines the number of lines of the body of an if / unless
+            needs to have to trigger this cop.
+        :param indentation_width:
+            No. of spaces to indent.
+        :param method_name:
+            Case of a method's name.
+            options: 'snake_case', 'camelCase'.
+        :param string_literals:
+            Use ' or " as string literals.
+            options: 'single_quotes', 'double_quotes'
+        :param variable_name:
+            Case of a variable's name.
+            options: 'snake_case', 'camelCase'.
+        :param max_class_length:
+            Max lines in a class.
+        :param class_length_count_comments:
+            Count comments while calculating class length?
+        :param max_module_length:
+            Max lines in a module.
+        :param module_length_count_comments:
+            Count comments while calculating module length?
+        :param cyclomatic_complexity:
+            Cyclomatic Complexity of the file.
+        :param max_line_length:
+            Max length of a line
+        :param line_length_allow_here_doc:
+            Allow here-doc lines to be more than Max in length
+        :param line_length_allow_uri:
+            To make it possible to copy or click on URIs in the code,
+            we allow ignore long lines containing a URI to be longer than Max.
+        :param max_method_length:
+            Max number of lines in a method.
+        :param method_count_comments:
+            Count full line comments?
+        :param max_parameters:
+            Max number of parameters in parameter list.
+        :param count_keyword_args:
+            Count keyword args while counting all arguments?
+        :param ignore_unused_block_args_if_empty:
+            Ignore unused block arguments if block is empty.
+        :param allow_unused_block_keyword_arguments:
+            Allow unused block keyword arguments.
+        :param ignore_unused_method_args_if_empty:
+            Allows unused method argument if method is empty.
+        :param allow_unused_method_keyword_args:
+            Allows unused keyword arguments in a method.
```

Comment:
quick thought: this is a pretty long list of stuff and the order doesn't seem clear to me. The following options are more or less alphabetically sorted. would it be possible/ make any sense to sort the params here (and probably in the function on top) or is there a reason for this particular order?